guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share.png"
+   retina = "/assets/images/guide/payments/receive/share@2x.png"
+   alt-text = "Screen showing the invoice with a share button"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share-sharesheet.png"
+   retina = "/assets/images/guide/payments/receive/share-sharesheet@2x.png"
+   alt-text = "Screen showing the invoice overlayed with a share sheet"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+</div>
+
+
+**Text**
+
+Sharing payment details as text is as straightforward as copying and pasting. It is also typical to provide the receiver with a button to conveniently share in other applications, with “share sheets,” which most mobile operating systems offer.
+
+**QR Codes**
+
+QR Codes are also a common way to exchange payment details when the two devices are near one another. It is not without its faults though, as some people may not be familiar with QR Codes, and lower-end phones may have difficulty reading them.
+
+Here are some constraints to consider when displaying QR Codes:
+
+- QR Codes with more blocks (modules) are challenging to read for devices with low-resolution cameras.
+- There should be sufficient contrast between the blocks in a QR Code and the background. Ideally, keeping the blocks black and background white would be the most optimal.
+- Size matters. You won't know the distance the devices will be from one another when the scan is happening, so you should design assuming the longest scan distance.
+
+## Waiting and transaction processing
+Once the payment details are shared, until the sender creates a transaction and broadcast it, there is uncertainty of when the payment would be completed. Think about how to keep the receiver informed while they are waiting.
+
+Similar to [the processes a sender](https://bitcoin.design/guide/payments/send/#transaction-processing-and-confirmation) sees, with the inclusion of this waiting period, here is what the receiver can expect:
+
+**Awaiting payment** -- the sender has not broadcasted a transaction with the payment, or one has not been detected as yet by the receiver's application.
+
+**Pending/Unconfirmed** -- miners know about the transaction, but have yet to process it. 
+
+**First confirmation** -- at this point the receiver may already be able to spend the funds, but has to be mindful of [chain reorganizations](https://bitcoin.design/guide/payments/transactions/#7-confirmations).
+
+**Sixth confirmation** -- from this point the payment is finalized, it should be no problem to spend freely.
+
+If your application has an area where the user can see a list of payment requests, this would be a good place to use it to also indicate which specific stage the payment is in.
```

Comment:
There are several screens with lists of transaction in the wallet UI kit we can use here. [Check it](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1339%3A32742).